app/controllers/shared/api.php

```diff
@@ -794,10 +810,16 @@
         }
     });
 
-App::init()
+Http::init()
     ->groups(['usage'])
     ->action(function () {
         if (System::getEnv('_APP_USAGE_STATS', 'enabled') !== 'enabled') {
             throw new Exception(Exception::GENERAL_USAGE_DISABLED);
         }
     });
+
+Http::shutdown()
+    ->inject('connections')
+    ->action(function (Connections $connections) {
+        $connections->reclaim();
+    });
```

Comment:
where was this code before? I guess in http.php right? any reasons  for moving it here?

Reply:
It's a new connections check [this](https://github.com/appwrite/appwrite/pull/7975#discussion_r1745904413)